src/map.cpp

```diff
@@ -7296,10 +7308,14 @@ void map::actualize( const tripoint &grid )
         for( int y = 0; y < SEEY; y++ ) {
             const tripoint pnt = sm_to_ms_copy( grid ) + point( x, y );
             const point p( x, y );
-            const auto &furn = this->furn( pnt ).obj();
+            const furn_t &furn = this->furn( pnt ).obj();
+            const ter_t &terr = this->ter( pnt ).obj();
             if( furn.has_flag( "EMITTER" ) ) {
```

Comment:
Could this be changed to `!furn.emissions.empty()`?

Reply:
Yes it can. Should we do it and get rid of this flag?

Reply:
Yeah, it looks redundant.